<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meet_the_team</class>
 <widget class="QDialog" name="meet_the_team">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-color: black;</string>
  </property>
  <widget class="QLabel" name="member1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(161, 129, 200);</string>
   </property>
   <property name="text">
    <string>Roxana</string>
   </property>
  </widget>
  <widget class="QLabel" name="member2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(161, 129, 200);</string>
   </property>
   <property name="text">
    <string>Eva</string>
   </property>
  </widget>
  <widget class="QLabel" name="member3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(161, 129, 200);</string>
   </property>
   <property name="text">
    <string>Maria</string>
   </property>
  </widget>
  <widget class="QLabel" name="member4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(161, 129, 200);</string>
   </property>
   <property name="text">
    <string>Daria</string>
   </property>
  </widget>
  <widget class="QLabel" name="nailslbl">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bodoni MT</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 255);</string>
   </property>
   <property name="text">
    <string>NAILS</string>
   </property>
  </widget>
  <widget class="QLabel" name="hairlbl">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bodoni MT</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 255);</string>
   </property>
   <property name="text">
    <string>HAIR</string>
   </property>
  </widget>
  <widget class="QLabel" name="cosmeticslbl">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bodoni MT</family>
     <pointsize>11</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 255);</string>
   </property>
   <property name="text">
    <string>COSMETICS</string>
   </property>
  </widget>
  <widget class="QLabel" name="member1_photo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0116.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member2_photo">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0114.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member3_photo">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0115.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member4_photo">
   <property name="geometry">
    <rect>
     <x>548</x>
     <y>30</y>
     <width>141</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0117.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member1_photo_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>330</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0116.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member3_photo_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>470</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0115.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member4_photo_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>610</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0117.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member1_photo_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>470</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0116.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member3_photo_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>620</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0115.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="member2_photo_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>101</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black; border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>static/IMG-20240730-WA0114.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bookApptBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>730</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 255);</string>
   </property>
   <property name="text">
    <string>Book an appointment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>760</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 255);</string>
   </property>
   <property name="text">
    <string>Go Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>